src/main/java/core/basesyntax/dao/FileReaderCsvImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderCsvImpl implements FileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        Path filePath = Paths.get(fileName);
+        List<String> lines;
+        try {
+            lines = Files.readAllLines(filePath);
+            lines.remove(0);
```

Comment:
you should do this in the method where you convert List<String> to List<Transaction>